packages/ui/tsconfig.json

```diff
@@ -8,9 +8,9 @@
     "importHelpers": true,
     "isolatedModules": true,
     "jsx": "react-jsx",
-    "lib": ["es2021.intl", "dom"],
-    "module": "NodeNext",
-    "moduleResolution": "NodeNext",
+    "lib": ["es6", "es2021.intl", "dom"],
+    "module": "es2020",
+    "moduleResolution": "Bundler",
```

Comment:
TypeScript complained about `import.meta` usage under the `NodeNext` module resolution algorithm. `Bundler` seems to be the right choice anyway since we're bundling with esbuild. I also had to change the `module` to something other than `NodeNext`.